rules/aws_write_only_attributes.go

```diff
@@ -0,0 +1,127 @@
+package rules
+
+import (
+	"fmt"
+
+	"github.com/terraform-linters/tflint-plugin-sdk/hclext"
+	"github.com/terraform-linters/tflint-plugin-sdk/tflint"
+	"github.com/terraform-linters/tflint-ruleset-aws/project"
+	"github.com/zclconf/go-cty/cty"
+)
+
+// AwsWriteOnlyAttributesRule checks if a write-only attribute is available for sensitive input attributes
+// It emits a warning if the normal attribute is used, as that is a non-ephemeral attribute and the secret value is stored in state
+type AwsWriteOnlyAttributesRule struct {
+	tflint.DefaultRule
+
+	writeOnlyAttributes map[string]writeOnlyAttribute
+}
+
+type writeOnlyAttribute struct {
+	original             string
+	writeOnlyAlternative string
+	otherAlternative     string
+}
+
+// NewAwsWriteOnlyAttributesRule returns new rule with default attributes
+func NewAwsWriteOnlyAttributesRule() *AwsWriteOnlyAttributesRule {
+	writeOnlyAttributes := map[string]writeOnlyAttribute{
+		"aws_secretsmanager_secret_version": {
```

Comment:
On first glance I notice `aws_db_instance` missing and that probably means plenty of other less common services are missing too.

You could generate these:

https://developer.hashicorp.com/terraform/cli/commands/providers/schema

Go through all resources looking for `*_wo`. Even if you don't actually add a generator here just doing it once at least guarantees your initial data is complete.

Reply:
I've generated both rule and test based on the schema 💪

Thanks for showing the way 🚀